src/pages/home/report/ReportActionCompose.js

```diff
@@ -437,8 +523,17 @@ class ReportActionCompose extends React.Component {
      * @param {Object} e
      */
     triggerHotkeyActions(e) {
-        // Do not trigger actions for mobileWeb or native clients that have the keyboard open because for those devices, we want the return key to insert newlines rather than submit the form
-        if (!e || this.props.isSmallScreenWidth || this.props.isKeyboardShown) {
+        if (e.key === 'Enter' && this.state.suggestedEmojis.length) {
+            this.insertSelectedEmoji(this.state.highlightedEmojiIndex);
+            e.preventDefault();
+            return;
+        }
+        if (e.key === 'Escape' && this.state.suggestedEmojis.length) {
+            this.setState({
+                suggestedEmojis: [],
+                shouldShowSuggestionMenu: false,
+            });
+            e.preventDefault();
```

Comment:
Is it potentially problematic that we're preventing the default _after_ calling setState? I haven't seen it cause problems ever, I'm just curious

Reply:
As long as the behavior is tested thoroughly, there should not be any problems with preventing the default behavior in this way.